validator/client/runner.go

```diff
@@ -35,11 +35,7 @@ var backOffPeriod = 10 * time.Second
 func run(ctx context.Context, v iface.Validator) {
 	cleanup := v.Done
 	defer cleanup()
-	if err := v.WaitForWalletInitialization(ctx); err != nil {
-		// log.Fatalf will prevent defer from being called
-		cleanup()
-		log.Fatalf("Wallet is not ready: %v", err)
-	}
+
```

Comment:
Why is this moved?

Reply:
the web3signer requires the genesis validators root that only happens after chain start